app/Annotations/OpenApi/controllersAnnotations/studentAnnotations/AnnotationsStudentDetail.php

```diff
@@ -5,66 +5,63 @@
 class AnnotationsStudentDetail
 {
     /**
-     * @OA\Get(
-     *     path="/student/detail/for-home",
-     *     operationId="getStudentDetail",
+     * @OA\Get (
+     *     path="/student/{id}/detail/for-home",
+     *     operationId="getStudentDetailsabout",
      *     tags={"Student"},
-     *     summary="Get Student Detail.",
-     *     description="Get student detail registered. No authentication required",
-     *
+     *     summary="Obtener detalles de un estudiante.",
```

Comment:
To maintain uniformity with the rest of the documentation, this type of annotations (summary, description, ...) should be in English.

Reply:
Done. Thank you very much.